server/datastore/mysql/migrations/tables/migration_test.go

```diff
@@ -0,0 +1,83 @@
+// Package tables holds fleet table migrations.
+//
+// Migrations can be tested with tests following the following format:
+//
+//	$ cat 20220208144831_AddSoftwareReleaseArchVendorColumns_test.go
+//
+//	[...]
+//	func TestUp_20220208144831(t *testing.T) {
+//		db := newDBConnForTests(t)
+//
+//		// Apply all migrations up to 20220208144831 (name of test), not included.
+//		applyUpToPrev(t, db)
```

Comment:
I suspect that pretty much all tests will follow that pattern, I think Tomas' comment to do it automatically as part of `newDBConnForTests` makes sense.